website/templates/coach_dashboard.html

```diff
@@ -1,10 +1,25 @@
-{% extends "base2.html" %}
+{% extends "base2.html" %} 
 
 
 {% block content %}
 
+<link rel="stylesheet" href="{{url_for('static', filename = 'assets/css/coach_dashbord.css')}}">
+<link rel="stylesheet" href="static/assets/css/coach_dashboard.css">
+
 
 <h1>{{ team.team_name }}</h1>
+<a class="nav-link1" id="profileDropdown1" href="#" data-toggle="dropdown">
```

Comment:
The dropdown menu you added shows up in the middle of the page. Can you change it so it is in the navigation bar?

Reply:
done